docs/concepts/overview.md

```diff
@@ -0,0 +1,32 @@
+---
+title:  "Quick Overview"
+weight: ?
```

Comment:
```suggestion
weight: 5
```

Reply:
This is related to how the pages are ordered in the sidebar, and it must be a number, I believe 0-100. 5 will work. (There's only one other page in the concepts folder right now.)